re/golang.go

```diff
@@ -33,7 +34,23 @@ func (re *re) processGo(ctx context.Context, details []slog.Attr, r Rule, msg *m
 	if err != nil {
 		return pkglog.RunInfo{Level: slog.LevelError, Details: details, Message: err.Error()}
 	}
-	for _, o := range r.Logic.Outputs {
+
+	var outputs []Runnable
+	for _, out := range r.Outputs {
+		b, e := json.Marshal(out)
+		if e != nil {
+			err = errors.Wrap(e, err)
+			continue
+		}
+		var o Output
+		if e := json.Unmarshal(b, &o); e != nil {
+			err = errors.Wrap(e, err)
+			continue
+		}
+		outputs = append(outputs, o.Runnable)
+	}
```

Comment:
`r.Outputs` is already  Runnable, that's all we care. There is no need for this JSON serialization and deserialization.